publiccode.yml

```diff
@@ -0,0 +1,62 @@
+publiccodeYmlVersion: "0"
```

Comment:
I'm sure this is a dumb question but why do we start with 0? :)

Reply:
You're right, the spec is currently at "v0.4.0".